src/main/java/com/ernie/TicketApp/service/TicketService.java

```diff
@@ -26,74 +26,80 @@ public static void main(String[] args) {
                 1
         );
 
-        Ticket ticketLimited = new Ticket(
-                "SecondHall",
-                888,
-                dateTime
-        );
-
-        Ticket ticketEmpty = new Ticket();
-
-        System.out.println(ticketFull);
-        System.out.println(ticketLimited);
-        System.out.println(ticketEmpty);
-
-        Client client = new Client();
-        client.printRole();
-        client.print();
-        client.getTicket(ticketLimited);
-
-        Admin admin = new Admin();
-        admin.checkTicket(ticketFull);
-
-        ticketEmpty.setId(null);
-        System.out.println(ticketEmpty.getId());
-        NullCheckUtility.validateNotNull(ticketEmpty);
-
-        ticketFull.shared("+7707777232424");
-        ticketFull.shared("+7707777232424", "some_email@mail.com");
-
-        System.out.println("Testing custom ArrayList");
-        MyArrayList myArrayList = new MyArrayList();
-        myArrayList.put("Element 1");
-        myArrayList.put("Element 2");
-        myArrayList.put("Element 3");
-        myArrayList.put("Element 4");
-        myArrayList.put("Element 5");
-        myArrayList.put("Element 6");
-        myArrayList.put("Element 7");
-        myArrayList.put("Element 8");
-        myArrayList.put("Element 9");
-        myArrayList.put("Element 10");
-        myArrayList.put("Element 11");
-
-        myArrayList.printAll();
-        System.out.println(myArrayList.size() + " " + myArrayList.capacity() + "\n");
-        System.out.println(myArrayList.getByIndex(4) + "\n");
-
-        myArrayList.deleteByIndex(4);
-        myArrayList.printAll();
-        System.out.println(myArrayList.size() + " " + myArrayList.capacity() + "\n");
-
-        myArrayList.print();
-
-        System.out.println("Testing custom HashSet");
-        MyHashSet myHashSet = new MyHashSet();
-        myHashSet.put("Element 1");
-        myHashSet.put("Element 2");
-        myHashSet.put("Element 3");
-        myHashSet.put("Element 3");
-        myHashSet.put("Element 3");
-
-        myHashSet.print();
-
-        System.out.println(myHashSet.contains("Element 4"));
-        System.out.println(myHashSet.contains("Element 2"));
-
-        myHashSet.delete("Element 1");
-
-        myHashSet.print();
+        ticketFull.setTicketType(TicketType.DAY);
```

Comment:
In order to avoid this mess in TicketService class, create separate methods or classes with methods for each HW (OOP, Collections, JDBC etc.) and call them from `main(...)` method. Thus, your app will be extended each time and previous implementation will not be modified.

Reply:
fixed everything in https://github.com/my20yeezy/Learning-at-Andersen/pull/4/commits/4a1eea5e3c36bf0f9fc08d01a951c451dcf23aef